q2_diversity/plugin_setup.py

```diff
@@ -655,7 +655,7 @@
                 'the same; the distance matrices will be reordered before '
                 'applying the Mantel test.\n\nSee the scikit-bio docs for '
                 'more details about the Mantel test:\n\n'
-                'http://scikit-bio.org/docs/latest/generated/generated/'
+                'http://scikit-bio.org/docs/latest/generated/'
                 'skbio.stats.distance.mantel.html',
```

Comment:
What do you think about dropping the `.html` from the end of the link?  The CLI chops the link up across two lines, making it unclickable, at least for me in iTerm2:

![image](https://user-images.githubusercontent.com/11708906/84819526-037e8080-afcd-11ea-8572-b64162383793.png)

http://scikit-bio.org/docs/latest/generated/skbio.stats.distance.mantel still works and fits on a single line.

```suggestion
                'http://scikit-bio.org/docs/latest/generated/'
                'skbio.stats.distance.mantel',
```